htop.c

```diff
@@ -35,37 +35,28 @@ in the source distribution for its full text.
 #include "UsersTable.h"
 #include "XUtils.h"
 
-#ifdef HAVE_LIBCAP
-#include <sys/capability.h>
-#endif
 
+#ifndef PLATFORM_LONG_OPTIONS
```

Comment:
My preference would be to not default-define them here, but define them in any `${platform}/platform.h` file.
That way we would fail hard on a missing definition or a messed up include order.